app/Http/Controllers/Assets/BulkAssetsController.php

```diff
@@ -53,6 +53,14 @@ public function edit(Request $request) : View | RedirectResponse
 
         $asset_ids = $request->input('ids');
         if ($request->input('bulk_actions') === 'checkout') {
+            $undeployable_assets = Asset::whereIn('id', $asset_ids)
+                ->whereHas('assetStatus', function ($query){
+                    $query->where('deployable', 0);
```

Comment:
I think we now allow pending items to be checked out as well

Reply:
updated to match the undeployable scope.